packetbeat/docs/fields.asciidoc

```diff
@@ -6,7 +6,7 @@ This file is generated! See etc/fields.yml and scripts/generate_field_docs.py
 [[exported-fields]]
 == Exported Fields
 
-This document describes the fields that are exported by Packetbeat. They are
+This document describes the fields that are exported by Docs/Fields.Asciidoc. They are
```

Comment:
Me either. :confused:  I used `make update` re-build the Packetbeat docs because I added a field. I'll open a new PR to undo this change.